includes/class-llms-block-templates.php

```diff
@@ -0,0 +1,663 @@
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'archive-llms_membership'    => esc_html__( 'Membership Catalog', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'single-certificate'         => esc_html__( 'Single Certificate', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'single-no-access'           => esc_html__( 'Single Access Restricted', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-course_cat'        => esc_html__( 'Taxonomy Course Category', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-course_difficulty' => esc_html__( 'Taxonomy Course Difficulty', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-course_tag'        => esc_html__( 'Taxonomy Course Tag', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-course_track'      => esc_html__( 'Taxonomy Course Track', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-membership_cat'    => esc_html__( 'Taxonomy Membership Category', 'lifterlms' ),
+			self::LLMS_BLOCK_TEMPLATES_PREFIX . 'taxonomy-membership_tag'    => esc_html__( 'Taxonomy Membership Tag', 'lifterlms' ),
+		);
+
+		/**
+		 * Filters the block template titles.
+		 *
+		 * @since [version]
+		 *
+		 * @param array $template_titles  {
+		 *     Associative array of template titles.
+		 *
+		 *     @type string $slug  The template slug.
+		 *     @type string $title The template readable titles.
+		 * }
+		 */
+		return apply_filters( 'lifterlms_block_templates_titles', $template_titles );
+
+	}
+
+	/**
+	 * Block Templates admin js strings.
+	 *
+	 * @since [version]
+	 *
+	 * @return string[]
+	 */
+	private function block_editor_l10n() {
+
+		return array(
+			'archive-course'             => esc_html__( 'LifterLMS Course Catalog Template', 'lifterlms' ),
+			'archive-llms_membership'    => esc_html__( 'LifterLMS Membership Catalog Template', 'lifterlms' ),
+			'single-certificate'         => esc_html__( 'LifterLMS Certificate Template', 'lifterlms' ),
+			'single-no-access'           => esc_html__( 'LifterLMS Single Template Access Restricted', 'lifterlms' ),
+			'taxonomy-course_cat'        => esc_html__( 'LifterLMS Course Category Taxonomy Template', 'lifterlms' ),
+			'taxonomy-course_difficulty' => esc_html__( 'LifterLMS Course Difficulty Taxonomy Template', 'lifterlms' ),
+			'taxonomy-course_tag'        => esc_html__( 'LifterLMS Course Tag Taxonomy Template', 'lifterlms' ),
+			'taxonomy-course_track'      => esc_html__( 'LifterLMS Course Track Taxonomy Template', 'lifterlms' ),
+			'taxonomy-membership_cat'    => esc_html__( 'LifterLMS Membership Tag Taxonomy Template', 'lifterlms' ),
+			'taxonomy-membership_tag'    => esc_html__( 'LifterLMS Membership Tag Taxonomy Template', 'lifterlms' ),
+		);
+
+	}
+
+	/**
+	 * Localize block templates.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function localize_blocks() {
+		wp_localize_script(
+			'llms-blocks-editor',
+			'LLMS_Block_Templates_l10n',
```

Comment:
I know that in a lot of the codebase we use phpcs standards for JS and this kind of stuff but as since this will ultimately be a JS variable we should make it a JS-style variable.

```suggestion
			'llmsBlockTemplatesL10n',
```